### ISSUE TITLE
preact is rendering boolean values as string

### ISSUE THREAD

**Issue by maxguzenski** [developit/preact#433]
Preact is rendering boolean value as string (I'm using preact-compat), React render as a empty node. 

```jsx
<div>{true}</div>

//output with preact
<div>true</div>

//output with react
<div></div>
```

**Comment by developit** [comment]
Wow, I had no idea that was how React handled `true`. Seems strange to be honest, but I also can't think of a time I've intentionally rendered `true` as a Boolean Text node.  I think we can switch preact to match React here.

Demo:
https://jsfiddle.net/developit/981Lqu6s/

**Comment by maxguzenski** [comment]
I know that this is not best solution, but my App has a lot of this:

```jsx
const MenuItem = (props) => (
  <Row> 
    { props.leftIcon && <Col>{ props.leftIcon }</Col> }
    <Col>{ props.primaryText }</Col>
  </Row>
)

<Menu>
  <MenuItem leftIcon={<Icon icon='star' />} text='star' />
  <MenuItem leftIcon={true} text='I want just a empty space to icon here, to align with prev menu' />
</Menu>
```

**Comment by developit** [comment]
ahh interesting. Usually you'd use a string for that: `leftIcon=""`.  I'm fine adding support for the true bypass though, might actually save a couple bytes.

**Comment by jxodwyer** [comment]
Hey @maxguzenski  @developit  I just pushed up a PR for this: https://github.com/developit/preact/pull/468

**Comment by developit** [comment]
Fixed in [7.2.0](https://github.com/developit/preact/releases/tag/7.2.0) (beta)!
